Lib/pickle.py

```diff
@@ -1049,14 +1048,11 @@ def _batch_appends(self, items, obj):
                 write(APPENDS)
             elif n:
```

Comment:
This can be replaced with `else`, since `batched()` does not emit empty batches.